### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by normallytangent**
modified the URL to HTTP from HTTPS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
-You can play the game [here](https://Chelwin.github.io/github-games/)
+You can play the game [here](http://Chelwin.github.io/github-games/)
```
Comment (normallytangent) [inline README.md:5]: modified since https is not enabled on my website

**Review by normallytangent — COMMENTED** [review]
- changed https to http
- modified since https is not enabled on my website

**Review by ppremk — APPROVED** [review]
👍

**Review by ppremk — APPROVED** [review]

**Comment by ppremk** [comment]
🎉 it's approved now 🙂

**Comment by normallytangent** [comment]
Yay! :D Thank you for your help!

**Pull request merged**